nexus/external-api/src/lib.rs

```diff
@@ -3660,79 +3693,48 @@ pub trait NexusExternalApi {
     /// queued for re-delivery.
     #[endpoint {
         method = POST,
-        path = "/v1/webhooks/receivers/{receiver}/probe",
-        tags = ["system/webhooks"],
+        path = "/v1/webhook-receivers/{receiver}/probe",
+        tags = ["system/alerts"],
     }]
     async fn webhook_receiver_probe(
```

Comment:
is this particular to webhook receivers? In the CLI we had decided to call this subcommand `oxide alert receiver probe` i.e. assuming it would not be specific to webhooks but general for all receivers.

Reply:
it returns a response model that is webhook-specific (it contains information about the status code of the HTTP response etc).

we could make that model be an enum with a single variant if we think it's preferable for this to be a route that applies to all receiver types. but, my thinking was that it's possible there might be some future receiver types that cannot be probed (can you synchronously probe an email?)

Reply:
Changed this to not be webhook-specific in eae40e0716a7ea376708fb15c13efe648722d925.